test/helper/with_term.rb

```diff
@@ -1,5 +1,3 @@
-require 'stringio'
```

Comment:
I don't want to remove this `require` because this file is independent of `unicode_plot`.

Reply:
I added this line back in.